### ISSUE TITLE
HotDog Classification

### ISSUE THREAD

**Issue by abhisheks008** [abhisheks008/DL-Simplified#27]
### Deep Learning Simplified Repository (Proposing new issue)
:red_circle: **Project Title** : HotDog Classification</br>
:red_circle: **Aim** : Aim is to identify the images which are hotdog or not a hotdog.</br>
:red_circle: **Dataset** : https://www.kaggle.com/datasets/thedatasith/hotdog-nothotdog</br>
:red_circle: **Approach** : Try to use 3-4 algorithms to implement the models and compare all the algorithms to find out the best fitted algorithm for the model by checking the accuracy scores. Also do not forget to do a exploratory data analysis before creating any model.

**********************************************************************************
### 📍 **Follow the Guidelines to Contribute in the Project :**
* You need to create a separate folder named as the Project Title.
* Inside that folder, there will be four main components.
   * Images - To store the required images.
   * Dataset - To store the dataset or, information/source about the dataset.
   * Model - To store the machine learning model you've created using the dataset.
   * `requirements.txt` - This file will contain the required packages/libraries to run the project in other machines.
* Inside the `Model` folder, the `README.md` file must be filled up properly, with proper visualizations and conclusions.

***********************************************************************
:red_circle::yellow_circle: **Points to Note :**

- The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.
- "Issue Title" and "PR Title should be the same. Include issue number along with it.
- Follow Contributing Guidelines & Code of Conduct before start Contributing.

***********************************************************************
:white_check_mark: **To be Mentioned while taking the issue :**
- Full name : 
- GitHub Profile Link : 
- Email ID :
- Participant ID (if applicable):
- Approach for this Project :
- What is your participant role? (Mention the Open Source program)

*************************************************************
Happy Contributing 🚀 

All the best. Enjoy your open source journey ahead. 😎

**Comment by Vikalp981** [comment]
Hey, @abhisheks008, I would like to work on this issue.

Full name: Vikalp Kumar Tripathi
GitHub Profile Link: https://github.com/Vikalp981
Email ID: tripathivikalp862@gmail.com
Approach for this Project: First, I am thinking of using a Convolutional Neural Network as a direct way of doing it, and then another approach I will use to solve this will be classifying it using transfer learning. In another way, I will use Pytorch Deep Learning Framework and then compare the accuracy of all of them.
Participant role : SSOC 2022

**Comment by LahariBoni** [comment]
Assigned it to you @Vikalp981, your approach sounds great. Go ahead.
